source/_integrations/roku.markdown

```diff
@@ -8,16 +8,26 @@ ha_category:
   - Remote
 ha_iot_class: Local Polling
 ha_release: 0.86
+ha_config_flow: true
+ha_quality_scale: silver
 ---
 
-The [Roku](https://www.roku.com/) integration allows integration of Roku, which will be automatically discovered if you enable the [discovery component](/integrations/discovery/).
+The `roku` platform allows you to control a [Roku](https://www.roku.com/) device.
+
+### Setup
+
+Go to the integrations page in your configuration and click on new integration -> Roku.
+If you have enabled [ssdp](/integrations/ssdp) discovery and your Roku device is on, it's likely that you just have to confirm the detected device.
 
 There is currently support for the following device types within Home Assistant:
 
 - Media Player
 - Remote
 
-The `roku` integration can also be forced to load by adding the following lines to your `configuration.yaml`:
+### YAML Configuration
+
+YAML configuration is around for people that prefer YAML.
```

Comment:
Yeah, let's remove this line. Its a bit... Dunno, Pepsi is around for people that prefer Pepsi... 😉 
```suggestion
YAML configuration is around for people that prefer YAML.
```